packages/connector-aws-ses/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "logto-connertor-aws",
```

Comment:
In order to be recognized by Logto's core, the name should follow the format which is `@logto/connector-xxx`.